### DISCUSSION TITLE
Problem with `ui.textarea()` Zoom on Mobile Devices in NiceGUI Web-App

### DISCUSSION THREAD

**Discussion by LTaxus** [zauberzeug/nicegui#2408] [Q&A]
When using `ui.textarea()` in my NiceGUI web application, I encounter an issue on mobile devices. Tapping on the textarea causes the view to automatically zoom in on the field. This necessitates manually zooming out to view the rest of the page again. This behavior does not occur on desktop devices. The automatic zoom disrupts the user experience, especially on smaller screens.

Is there a possibility to influence the behavior?

**Comment by falkoschindler** [comment]
Hi @LTaxus,
I can't immediately reproduce the behavior. But I guess it's a general problem with web applications in general, not only NiceGUI. I asked ChatGPT and it came up with the following ideas (among others):
1. Try running with `ui.run(..., viewport='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no')`.
2. Use a larger font size for the text area so that zooming is not necessary.

**Reply by LTaxus** [reply]
Hi @falkoschindler 
thank you for your respond and the suggestions. I will try them and come back.

**Reply by LTaxus** [reply]
Hi @falkoschindler 

The first one with `viewport` has no effect on this behavior. I also tried to change the text size of my smartphones settings and I experienced the same "zoom" behavior.

Your second suggestion worked well! Thank you and ChatGPT 😅. When I was asking, it didn't came up with this.

I played a bit with the text size and it turned out with `text-base` the behavior disappeared but still with the next smaller one (`text-sm`). Seems like `text-sm` is the default size of `ui.textarea()`?

Regards

**Reply by falkoschindler** [reply]
> Seems like `text-sm` is the default size of `ui.textarea()`?

Possibly. NiceGUI doesn't change the default text size of the underlying [QInput](https://quasar.dev/vue-components/input) component.
